section2/exercises/decision-making.js

```diff
@@ -38,16 +38,29 @@ if (bearChoice === 1) {
 Questions
 
 1. In English, using technical vocabulary, describe what is happening between lines 14 and 18.
```

Comment:
Really nice job with these - spot on, and I appreciate the level of detail you went into.